roles/loki/vars/main.yml

```
@@ -32,6 +32,16 @@ _loki_helm_values:
         installOperator: false
     lokiCanary:
       enabled: false
+    rules:
+      additionalGroups:
+        - name: additional-loki-rules
+          rules:
+            - alert: NovaCellNotResponding
+              expr: 'kubernetes.labels.component="compute" and log =~ "not responding and hence is being omitted from the results"'
```

```suggestion
              expr: 'kubernetes.labels.component="compute" and log =~ /nova\.compute\.multi_cell_list.*Cell [0-9a-fA-F-]+ is not responding/'
```
I think we can use regex here but it will be more or less resource-intensive.